### DISCUSSION TITLE
language selection for each user

### DISCUSSION THREAD

**Discussion by rjv23** [Leantime/leantime#495] [Ideas]
hi, just found the app and am very impressed with features and ease of use so far. 
Is it possible to configure so each user can chose their language for menus, etc. of course input language would be in user selected, but in an multi-lingual company having choice of language is quite important. tks for your comments and all the hard work on the app so far

**Comment by marcelfolaron** [comment]
I think that is a great idea and should be pretty simple to implement. Let me check

**Reply by PerseusAT** [reply]
Hello.
Is there anything new about this? Very important in some multi culture company. i miss this function.

**Comment by rjv23** [comment]
tks for your reply, appreciate your hard work on the app. 
     On Friday, November 5, 2021, 11:43:13 PM GMT+8, Marcel Folaron ***@***.***> wrote:  
 
 


I think that is a great idea and should be pretty simple to implement. Let me check

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
